### DISCUSSION TITLE
Enhancing Insight: Implementing Repository Views Counters

### DISCUSSION THREAD

**Discussion by 03prashantk** [yhype/yhype-feedback#28] [Q&A] [answered]
Certainly! We can embark on developing a repository views counter, leveraging the repository names as unique identifiers. This initiative will enable us to gain valuable insights into the accessibility and popularity of our repositories among our audience. By tracking views over time, we can better understand user engagement patterns, identify trends, and tailor our strategies to meet the evolving needs of our community.

**Comment by antonkomarev** [comment, marked as answer]
But we already have repos views count, and even more, we have unique visitors count.

Here is a real example of one of my repositories Traffic page.

![Screenshot_2024-04-16-21-11-29-132_com.android.chrome.jpg](https://github.com/yhype/yhype-feedback/assets/1849174/4063ccf5-f3b7-4aba-af78-97be8829564c)

**Reply by 03prashantk** [reply]
Oh great I didn't knew this... And BTW can we embed this on our each GitHub repo too?

**Reply by antonkomarev** [reply]
You don't need to. Just choose what repositories you want to track in your YHYPE account. Ot is working out of the box.
